CMakeLists.txt

```diff
@@ -478,7 +472,7 @@ if(${INSTALL_PYTHON_FILES} MATCHES ON)
   set(CPACK_PACKAGE_FILE_NAME  "Kratos_${KratosMultiphysics_VERSION_MAJOR}_${KratosMultiphysics_VERSION_MINOR}")
   #SET(CPACK_DEBIAN_PACKAGE_SHLIBDEPS ON)
   SET(CPACK_DEBIAN_PACKAGE_DEPENDS ${CPACK_DEBIAN_PACKAGE_DEPENDS} "libc6, libgcc1, libgfortran3, libgomp1, libopenmpi1.3, libparmetis3.1, libstdc++6, openmpi-bin, ${KRATOS_EXTRA_PACKAGE_DEPENDS} " )
-  #    SET(CPACK_DEBIAN_PACKAGE_DEPENDS ${CPACK_DEBIAN_PACKAGE_DEPENDS} " openmpi-bin, libboost-mpi-python-dev, ${KRATOS_EXTRA_PACKAGE_DEPENDS}" )
+  #    SET(CPACK_DEBIAN_PACKAGE_DEPENDS ${CPACK_DEBIAN_PACKAGE_DEPENDS} " openmpi-bin, ${KRATOS_EXTRA_PACKAGE_DEPENDS}" )
```

Comment:
why not deleting the entire line?

Reply:
I have never used the Debian packaging, so I thought I would just remove the package that is no longer needed... I agree that CMakeLists should get a nice cleanup at some point, though...

Reply:
I would go for this cleanup in another Pr